app/assets/javascripts/app/components/comments_list/comments_list.js

```diff
@@ -3,48 +3,16 @@
 
   angular
     .module('council.components')
-    .directive('commentsList', CommentsList)
-    .controller('CommentsListCtrl', CommentsListCtrl);
+    .directive('commentsList', CommentsList);
 
   function CommentsList() {
     return {
       scope: {
         comments: '=commentsList',
-        count: '=commentCount'
+        count: '=commentCount',
+        toggle: '=toggle'
       },
-      controller: 'CommentsListCtrl as ctrl',
```

Comment:
I'm not sure it works or not. But you do not have a controller in this directive. That shouldn't be a cue to go back to using `$scope`again.